libdnf/base/goal.cpp

```diff
@@ -112,12 +116,16 @@ bool is_unique(
         if (element.size() != new_element_size) {
```

Comment:
Please pass vectors by reference (do not copy).
```
const std::vector<std::vector<std::pair<ProblemRules, std::vector<std::string>>>> & problems,
const std::vector<std::pair<ProblemRules, std::vector<std::string>>> & new_element)
```

Reply:
Fixed